WordPress 5.x/WordPress 5.3/02-troubleshooting.md

```diff
@@ -6,8 +6,8 @@ Before posting, please make sure you’ve tried started by performing the troubl
 - **Troubleshooting with your browser**. Your browser can help you identify JavaScript issues or conflicts and [this article can assist you in doing that diagnosis](https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/). This could help identify Visual Editor issues as well.
 - **Make sure you have the Visual Editor enabled**. Visit your Users->Your Profile page. The first option will disable the visual editor. Make sure that option is unchecked, and save your profile settings.
 - Deactivating **all plugins** (yes, all) to see if this resolves the problem. If this works, re-activate the plugins one by one until you find the problematic plugin(s). If you can’t get into your admin dashboard, try resetting the plugins folder by [SFTP/FTP](https://wordpress.org/support/article/ftp-clients/) or PhpMyAdmin (read [“How to deactivate all plugins when you can’t log in to wp-admin”](https://wordpress.org/support/article/faq-troubleshooting/) if you need help). Sometimes, an apparently inactive plugin can still cause problems. Also remember to deactivate any plugins in the mu-plugins folder. The easiest way is to rename that folder to `mu-plugins-old`
-- Switching to the Twenty Nineteen theme to rule out any theme-specific problems. If you can’t log in to change themes, you can remove the theme folders via [SFTP/FTP](https://wordpress.org/support/article/ftp-clients/) so the only one is `twentynineteen`. That will force your site to use it.
-- Manually upgrading. When all else fails, download a fresh copy of the latest.zip file of 5.3.* (top right on this page) to your computer, and use that to copy up. You may need to delete the wp-admin and wp-includes folders on your server (NOTE: do not delete the **wp-content** directory or your **wp-config.php** file) **Read the [Manual Update directions first](https://wordpress.org/support/article/updating-wordpress/)**.
 - If you can install plugins, install “Health Check”: https://wordpress.org/plugins/health-check/. On the troubleshooting tab, you can click the button to disable all plugins and change the theme for you, while you’re still logged in, **without affecting normal visitors to your site**.
+- Switching to the Twenty Twenty theme to rule out any theme-specific problems. If you can’t log in to change themes, you can remove the theme folders via [SFTP/FTP](https://wordpress.org/support/article/ftp-clients/) so the only one is `twentytwenty`. That will force your site to use it.
```

Comment:
I'd retain Twenty Nineteen, we always wait one major release before changing this so that the default theme has a chance to mature, as there's historically often multiple fixes right after a new one is launched.

Reply:
OK, I wasn't aware... it's back to 2019